src/Actions/AssignUsersToWorkflowAction.php

```diff
@@ -20,6 +20,9 @@
  * @license    BSD License (http://silverstripe.org/bsd-license/)
  * @package    advancedworkflow
  * @subpackage actions
+ *
+ * @method \SilverStripe\ORM\ManyManyList<Member> Users()
+ * @method \SilverStripe\ORM\ManyManyList<Group> Groups()
```

Comment:
Should import ManyManyList rather than have the full FQCN here

Reply:
That's not how we did them previously, but I'll make the change to avoid ping pong (and cause I happen to prefer that anyway)

Done